templates/web/pages/product/product_page.tt.html

```diff
@@ -47,7 +47,7 @@
     <p>→ <a href="[% public_product_url %]">[% lang("product_page_on_the_public_database") %]</a></p>
 [% END %]
 
-[% IF (user_id.defined) && (robotoff_url.defined) && (robotoff_url.length > 0) %]
+[% IF (robotoff_url.defined) && (robotoff_url.length > 0) %]
```

Comment:
Maybe it could it a good idea to add a comment in the code to explain this

Reply:
done.